donkeycar/pipeline/types.py

```diff
@@ -32,36 +32,41 @@ def __init__(self, config: Any, base_path: str, underlying: TubRecordDict) -> No
         self.underlying = underlying
         self._image: Optional[Any] = None
 
-    def image(self, cached=True, normalize=False) -> np.ndarray:
-        if not self._image:
+    def image(self, cached=True) -> np.ndarray:
+        if self._image is None:
```

Comment:
Is this actually different from `if not self._image`?

Reply:
Yeah it is - if the image is an np.ndarray, then this tries to compare a single bool on the LHS against 'the truth value of an array' on  and falls over.

Reply:
:facepalm:  

Thanks for the explanation.